style.css

```diff
@@ -622,3 +625,128 @@ body {
   line-height: 24px;
   color: #000;
 }
+
+/* contact me */
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 48rem;
+  width: 375px;
```

Comment:
- [OPTIONAL] Avoid setting fixed width to the website element, since you are practicing mobile-first design approach all element width suppose to fill the whole screen width 👍🏼